packages/plugins/src/cli/install/index.ts

```diff
@@ -1,15 +1,28 @@
-import { StdioOptions, execSync } from 'child_process'
+import { execSync } from 'child_process'
 
 import { CONTRACTS_LIBRARY_COMMIT_HASH } from '@sphinx-labs/contracts'
+import { spawnAsync } from '@sphinx-labs/core'
 
-export const handleInstall = async (noCommit: boolean, stdio: StdioOptions) => {
+export const handleInstall = async () => {
   const args = [
-    'forge',
     'install',
     `sphinx-labs/sphinx@${CONTRACTS_LIBRARY_COMMIT_HASH}`,
+    '--no-commit',
```

Comment:
After testing, I found that you always have to use `--no-commit` during the installation process since installing our package results in there being changes in the users repo which would cause this to fail without `--no-commit`. So to prevent user error, I just made it the default.